### PR TITLE
rating component added

### PR DESCRIPTION
**Pull request by pariyar07**
Added 4 types of ratings

### CODE REVIEW

File: styles/rating.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    font-size: 1.5rem;
+}
+
+.custom-rating{
+    display: flex;
+    flex-direction: row;
+    list-style: none;
+    gap: 2rem;
+    font-size: 1.5rem;
+}
```
Comment (Tanay-J) [inline styles/rating.css]: Please use single class for set of properties which are being used multiple times, so as to avoid code repetition.

**Review by Tanay-J — CHANGES_REQUESTED** [review]

File: styles/rating.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    gap: 3rem;
+    width: 900px;
+    margin-left: 5rem;
+}
+
+.rating-container{
+    border: 1px solid white;
+    padding: 1rem 2rem;
+    border-radius: 8px;
+}
```
Comment (yogesh-) [inline styles/rating.css:15]: use color variables

File: styles/rating.css [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+.empty-rating, .filled-rating, .custom-rating, .emoji-rating{
```
Comment (yogesh-) [inline styles/rating.css:17]: make a class of flex and use that in all the divs where it is required, much faster than this.

**Review by yogesh- — APPROVED** [review]
Looks good, few minor changes suggested

**Pull request merged**